test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/RoutingTestCases.json

```diff
@@ -189,7 +208,8 @@
     "path": "/MinimalApi/123",
     "expectedStatusCode": 200,
     "currentHttpRoute": null,
-    "expectedHttpRoute": "/MinimalApi/{id}"
+    "currentMetricRoute": "/MinimalApi/{id}",
+    "expectedHttpRoute": "MinimalApi/{id}"
```

Comment:
What happens if a minimal api route has a parameter called `action`? Will it be rewritten? There is nothing stopping a minimal API having a parameter with that name and it isn't a static part of the route.